bilge-impl/src/shared/util.rs

```diff
@@ -0,0 +1,42 @@
+use syn::Path;
+
+pub trait PathExt {
```

Comment:
you could try pushing this to syn, either it will be "no" or it will be "hey you can already do this" or "thank you let's add it"

Reply:
I'll give it a try 🙂